Formula/genstats.rb

```diff
@@ -1,8 +1,8 @@
 class Genstats < Formula
   desc "Generate statistics about stdin or textfiles"
-  homepage "https://www.vanheusden.com/genstats/"
-  url "https://www.vanheusden.com/genstats/genstats-1.2.tgz"
-  sha256 "f0fb9f29750cdaa85dba648709110c0bc80988dd6a98dd18a53169473aaa6ad3"
+  homepage "https://web.archive.org/web/20180831170016/https://www.vanheusden.com/genstats/"
+  url "https://web.archive.org/web/20150331055106/http://vanheusden.com/genstats/genstats-1.2.tgz"
```

Comment:
```suggestion
  homepage "https://web.archive.org/web/20180831170016/www.vanheusden.com/genstats/"
  url "https://web.archive.org/web/20150331055106if_/vanheusden.com/genstats/genstats-1.2.tgz"
```

The `if_` part appears to be required.